src/refactorings/convert-for-to-foreach/convert-for-to-foreach.test.ts

```diff
@@ -180,6 +265,44 @@ items.forEach(item => {
         description: "for-loop with init not starting at 0",
         code: `for (let i = 1; i < items.length; i++) {
   console.log(items[i]);
+}`
+      },
+      {
+        description: "for-of but a string",
+        code: `const str = 'abcde';
+  for (let character of str) {
+  console.log(character);
+}`
+      },
+      {
+        description: "for-of but an inline string",
+        code: `for (let character of 'abcde') {
+  console.log(character);
+}`
+      },
+      // 👇 These patterns could be converted (they support forEach) but are not implemented yet.
```

Comment:
I went for this, I don't like having TODO comments that aren't tied to a test. 

That's a nice comment to keep around, so we remember why features aren't supported. I won't create these issues yet, I'll rather wait for demand.

Reply:
Makes sense!